src/neo4j_graphrag/embeddings/openai.py

```diff
@@ -17,7 +17,7 @@
 
 from typing import Any
 
-from neo4j_graphrag.embedder import Embedder
```

Comment:
It's maybe a detail, for the other packages (`llm` and `retrievers`), the interface (base class) is in `base.py`. Is it worth following the same sort of convention?

Reply:
I was thinking that but at the same time we expect users to create custom embedders so I was thinking this is more explicit. Although following this convention could make it less confusing, wdyt?

Reply:
We also expect users to create custom retrievers, right? But maybe the 'base' convention is not a good one and we should move to something more explicit everywhere?